bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/src/test/java/com/google/cloud/bigtable/mapreduce/hbasesnapshots/TestImportHBaseSnapshotJob.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.mapreduce.hbasesnapshots;
+
+import static com.google.cloud.bigtable.mapreduce.hbasesnapshots.ImportJobCommon.SNAPSHOTNAME_KEY;
+import static com.google.cloud.bigtable.mapreduce.hbasesnapshots.ImportJobCommon.SNAPSHOT_RESTOREDIR_KEY;
+
+import com.google.cloud.bigtable.hbase.BigtableConfiguration;
+import com.google.cloud.bigtable.hbase.BigtableOptionsFactory;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.CommonConfigurationKeysPublic;
+import org.apache.hadoop.hbase.HBaseConfiguration;
+import org.apache.hadoop.hbase.HConstants;
+import org.apache.hadoop.hbase.client.ClusterConnection;
+import org.apache.hadoop.hbase.mapreduce.TableOutputFormat;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/** test driver function */
+public class TestImportHBaseSnapshotJob {
+
+  private Configuration conf;
+
+  @Before
+  public void setup() {
+    this.conf = HBaseConfiguration.create();
+  }
+
+  @Test
+  public void testSetConfFromArgs() {
+    Configuration conf = HBaseConfiguration.create();
```

Comment:
shouldnt this be create(false) to avoid accidentally pulling in an hbase-site.xml?

Reply:
done